app/code/core/Mage/Core/Model/File/Validator/Image.php

```diff
@@ -85,8 +85,13 @@ public function validate($filePath)
         list($imageWidth, $imageHeight, $fileType) = getimagesize($filePath);
         if ($fileType) {
             if ($this->isImageType($fileType)) {
-                /** if 'general/reprocess_images/active' false then skip image reprocessing. */
-                if (!Mage::getStoreConfigFlag('general/reprocess_images/active')) {
+                /** Check deprecated 'general/reprocess_images/active' for BC. If false then skip image reprocessing. */
+                if (Mage::getStoreConfig('general/reprocess_images/active') !== null) {
+                    $imageQuality = Mage::getStoreConfigFlag('general/reprocess_images/active') ? 100 : 0;
```

Comment:
Why not migrate the config to the new location here? And maybe also delete the old one?

Reply:
I think because there was no backend option for that, it was just an entry in the config xmls, which I don't think it's possible to migrate

Reply:
Right, I didn't notice.